lib/simp/beaker_helpers.rb

```diff
@@ -2,7 +2,7 @@ module Simp; end
 
 module Simp::BeakerHelpers
   require 'simp/beaker_helpers/version'
-  DEFAULT_PUPPET_AGENT_VERSION = '1.8.3'
+  DEFAULT_PUPPET_AGENT_VERSION = '1.10.2'
```

Comment:
@jeefberkey After a bit of going back and forth, it sounds like this should be whatever the *oldest* supported version by SIMP is. That way we won't start using fancy new features and have our modules die on old systems.